### DISCUSSION TITLE
id_rsa permission denied after user id change

### DISCUSSION THREAD

**Discussion by PeeBeerBeach** [drakkan/sftpgo#1551] [Q&A] [answered]
I  changed the user id in my docker compose file and now the container wont run anymore.
I can still start it when i uncomment the new user id (so reverting to old one).
But i would like to start the process with a different user id without deleting settings and users inside SFTPGO.

Here is the error i get in the log:

`2024-02-29T19:45:06.765 INF No host keys configured and "id_rsa" does not exist; try to create a new host key | sender=sftpd
2024-02-29T19:45:11.721 WRN error creating host key "id_rsa": open id_rsa: permission denied | sender=sftpd
2024-02-29T19:45:11.721 ERR could not start SFTP server: open id_rsa: permission denied | sender=service`

Is there any way to migrate to a new user id?

**Comment by drakkan** [comment]
Hello,

SFTPGo is attempting to create a new host key and does not have file system level permissions to write the host key to disk

**Reply by PeeBeerBeach** [reply]
Ok thanks. Is it trying to create the host key file inside of the docker filesystem or on the host filesystem?
What should be the folder on a debian system? 

is it /root/.ssh/ ?

**Reply by drakkan** [reply, marked as answer]
if you are using the default Docker images with the default configuration, host keys are written to `/var/lib/sftpgo`

**Comment by PeeBeerBeach** [comment]
Thanks that solved it
